Makefile

```diff
@@ -30,10 +30,14 @@ all: depend generate test build image ## runs test, build and image
 test: lint ## test trust
 	KUBEBUILDER_ASSETS=$(BINDIR)/kubebuilder/bin go test -v ./pkg/... ./cmd/...
 
-.PHONY: test
-lint:
+.PHONY: lint
+lint: depend helm-docs
 	./hack/verify-boilerplate.sh
 
+.PHONY: helm-docs
+helm-docs: # verify helm-docs
+	./hack/verify-helm-docs.sh
+
```

Comment:
nit (non-blocking): Could we use `make verify-helm-docs` here? I think it'd be a little clearer that it's a verification step and that nothing is built from that make target.